python/pyspark/mllib/tests.py

```diff
@@ -303,7 +309,8 @@ def test_classification(self):
         self.assertEqual(same_dt_model.toDebugString(), dt_model.toDebugString())
 
         rf_model = RandomForest.trainClassifier(
-            rdd, numClasses=2, categoricalFeaturesInfo=categoricalFeaturesInfo, numTrees=100)
+            rdd, numClasses=2, categoricalFeaturesInfo=categoricalFeaturesInfo, numTrees=10,
+            maxBins=4)
         self.assertTrue(rf_model.predict(features[0]) <= 0)
```

Comment:
There is, but you can pass "seed"  If it's hard to find a good seed, I'd recommend setting featureSubsetStrategy = "all" as well (to eliminate a bit of stochasticity)